dotCMS/src/main/java/com/dotmarketing/util/ExportStarterUtil.java

```diff
@@ -0,0 +1,300 @@
+
+
+                    
+                    
+                }
+                Logger.info(this, "writing : " + total + " records for " + clazz.getName());
+            }
+
+
+            /* Run Liferay's Tables */
+            /* Companies */
+            _list = ImmutableList.of(APILocator.getCompanyAPI().getDefaultCompany());
+
+            _writing = new File(outputDirectory,  Company.class.getName() + ".xml");
+            
+            BundlerUtil.objectToXML(_list, _writing);
+            
+
+
+            /* Users */
+            _list = APILocator.getUserAPI().findAllUsers();
+            _list.add(APILocator.getUserAPI().getDefaultUser());
+
+            _writing = new File(outputDirectory,  User.class.getName() + ".xml");
+            BundlerUtil.objectToXML(_list, _writing);
+
+            /* users_roles */
+            dc = new DotConnect();
+
+            /* counter */
+            dc.setSQL("select * from counter");
+            _list = dc.getResults();
+
+            _writing = new File(outputDirectory, "/Counter.xml");
+            BundlerUtil.objectToXML(_list, _writing);
+
+
+            /* image */
+            _list = ImageLocalManagerUtil.getImages();
+
+            /*
+             * The changes in this part were made for Oracle databases. Oracle has problems when
+             * getString() method is called on a LONG field on an Oracle database. Because of this,
+             * the object is loaded from liferay and DotConnect is not used
+             * http://jira.dotmarketing.net/browse/DOTCMS-1911
+             */
+
+
+            _writing = new File(outputDirectory, "/Image.xml");
+            BundlerUtil.objectToXML(_list, _writing);
+
+            /* portlet */
+
+            /*
+             * The changes in this part were made for Oracle databases. Oracle has problems when
+             * getString() method is called on a LONG field on an Oracle database. Because of this,
+             * the object is loaded from liferay and DotConnect is not used
+             * http://jira.dotmarketing.net/browse/DOTCMS-1911
+             */
+            dc.setSQL("select * from portlet");
+            _list = dc.getResults();
+            _writing = new File(outputDirectory,"/Portlet.xml");
+            BundlerUtil.objectToXML(_list, _writing);
+
+            
+            //backup content types
+            File file = new File(outputDirectory,  "ContentTypes-" + ContentTypeImportExportUtil.CONTENT_TYPE_FILE_EXTENSION);
+            new ContentTypeImportExportUtil().exportContentTypes(file);
+
+            file = new File(outputDirectory, "/WorkflowSchemeImportExportObject.json");
+            WorkflowImportExportUtil.getInstance().exportWorkflows(file);
+
+            file = new File(outputDirectory, "/RuleImportExportObject.json");
+            RulesImportExportUtil.getInstance().export(file);
+
+            Logger.info(this, "Finished createXMLFiles into " + outputDirectory);
+        } catch (Exception e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)